src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -531,7 +531,7 @@ export class QuickBar extends LitElement {
         }
 
         mwc-list-item.command-item {
-          text-transform: capitalize;
+          text-transform: none;
```

Comment:
Since `none` is the default value, I think this whole CSS block could be removed then or is another one then interfering if it not set explicitly?